.github/workflows/ci.yaml

```diff
@@ -30,11 +30,8 @@ jobs:
             - name: Run linters
               shell: bash --noprofile --norc -x -eo pipefail {0}
               run: |
-                $(exit $(go fmt -modfile=go_test.mod  ./... | wc -l))
-                go vet -modfile=go_test.mod ./... 
-                GOFLAGS="-mod=mod -modfile=go_test.mod" staticcheck ./...
-                find . -type f -name "*.go" | xargs misspell -error -locale US
-                golangci-lint run --timeout 5m0s ./jetstream/...
+                GOFLAGS="-mod=mod -modfile=go_test.mod" golangci-lint run -c .golangci.yaml --timeout 5m0s ./...
```

Comment:
Probably better to use `golangci-lint-action` instead of shell actions, so that PRs are properly annotated with lint failures.